CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Cromwell Change Log
 
+## 53 Release Notes
+
+### Martha V3 Support
+
+Cromwell now supports resolving DRS URIs through martha_v3. Update the `martha.url` found in the [filesystems config](https://cromwell.readthedocs.io/en/stable/filesystems/Filesystems/#overview) to
+point to martha_v3. More information on martha_v3 request and response schema can be found [here](https://github.com/broadinstitute/martha#martha-v3). 
```

Comment:
```suggestion
## 53 Release Notes

### Martha v3 Support

Cromwell now supports resolving DRS URIs through Martha v3. Update the `martha.url` found in the [filesystems config](https://cromwell.readthedocs.io/en/stable/filesystems/Filesystems/#overview) to
point to `/martha_v3`. More information on Martha v3 request and response schema can be found [here](https://github.com/broadinstitute/martha#martha-v3).
```
Style: consistently distinguish between the marketing name and URL location

Also, we should land a PR to remove _Currently is in development stage. Please do not use it._ 😄

Reply:
#2 done — https://github.com/broadinstitute/martha/pull/180